tests/codegen-llvm/annotate-moves/annotate-moves-disabled.rs

```diff
@@ -0,0 +1,33 @@
+//
+//@ compile-flags: -Copt-level=0 -g
+
+// Test that move/copy operations are NOT annotated when the flag is disabled
```

Comment:
Nit: Remove empty `//`

```suggestion
// Test that move/copy operations are NOT annotated when the flag is disabled
//@ compile-flags: -Copt-level=0 -g
```